index.js

```diff
@@ -2,14 +2,16 @@
 
 'use strict';
 
-if (hexo.config.nofollow && hexo.config.nofollow.enable) {
-  const config = hexo.config.nofollow;
+const config = Object.assign({
```

Comment:
you should initialize all variables, currently there is error from
https://github.com/hexojs/hexo-filter-nofollow/blob/6ca54a2ac4ca600c8a58bee1b1c30b4fc9899288/lib/filter.js#L48
`Cannot read property field of undefined`.

This is how I recently do it in my plugin:
https://github.com/curbengh/hexo-yam/blob/master/index.js

Reply:
A default `field` and `exclude` config is now included in `Object.assign`.